pages/ksphere/konvoy/latest/tutorials/update-worker-machines/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: layout.pug
+navigationTitle: Update the image and other properties of worker machines
+title: Update the image and other properties of worker machines
+menuWeight: 5
+excerpt: Learn how to update the image and other properties of worker machines
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+If your cluster workload changes, you can use a different instance type for your worker machines. To ensure your worker machines' operating system is up to date, you can use a different machine image that includes a more recent patch version of the operating system.
+
+By default, Konvoy groups your worker machines in the `worker` node pool. If you change some properties of the machines and apply the change, the machines may be destroyed and re-created, disrupting their running workloads.
+
+This tutorial describes how to update the properties of worker machines without disrupting your cluster workload. You will create a new node pool with up-to-date properties, move your workload, to it, from the `worker` nodepool, and finally scale down the `worker` node pool.
+
+Follow these steps:
+
+1. Use this command to list all node pools, and identify the node pool with worker machines:
+
+    ```bash
+    konvoy get nodepools
+    ```
+
+    **NOTE**: If your workers are grouped in the `worker` node pool, the default for a Konvoy cluster, skip this step. If your cluster uses a different worker node pool, use that node pool name instead of `worker` in the following steps.
```

Comment:
```suggestion
    <p class="message--note"><strong>NOTE: </strong>  If your workers are grouped in the <code>worker</code> node pool, the default for a Konvoy cluster, skip this step. If your cluster uses a different worker node pool, use that node pool name instead of <code>worker</code> in the following steps.</p>
```